.github/workflows/acctest-oidc.yaml

```diff
@@ -13,28 +13,46 @@ permissions:
   id-token: write
 
 jobs:
-  go-version:
+  oidc-check:
     runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.go-version.outputs.version }}
+    permissions:
+      contents: read
+      id-token: write
     steps:
-      - uses: actions/checkout@v2
+      - id: oidc-check
+        run: |
+          set -v 
+          if [ "${ACTIONS_ID_TOKEN_REQUEST_URL}" = "" ]; then
```

Comment:
shouldn't this be a double-equals?

Reply:
Single bracket and single equals keeps this posix-compliant fwiw, but happy to change if it reads better